sdk/communication/communication-jobrouter/samples-dev/ClassificationPolicyCrud.ts

```
@@ -0,0 +1,210 @@
+      bypassSelectors: false
+    },
+    offerTtlSeconds: 15
+  };
+  await routerAdministrationClient.createDistributionPolicy(distributionPolicyRequest.id!, distributionPolicyRequest);
+
+  // define exception trigger for queue over flow
+  const queueLengthExceptionTrigger: QueueLengthExceptionTrigger = {
+    kind: "queue-length",
+    threshold: 100
+  }
+
+  const exceptionPolicyRequest: ExceptionPolicy = {
+    id: "exception-policy-123",
+    name: "test-policy",
+    exceptionRules: {
+      MaxWaitTimeExceeded: {
+        actions: {
+          MoveJobToEscalatedQueue: {
+            kind: "reclassify",
+            classificationPolicyId: "Main",
+            labelsToUpsert: {
+              escalated: true
+            }
+          }
+        },
+        trigger: queueLengthExceptionTrigger
+      }
+    }
+  };
+  await routerAdministrationClient.createExceptionPolicy(exceptionPolicyRequest.id!, exceptionPolicyRequest);
+
+  const queueRequest: JobQueue = {
+    id: "queue-123",
+    distributionPolicyId: "distribution-policy-123",
+    name: "Main",
+    labels: {},
+    exceptionPolicyId: "exception-policy-123"
+  };
+  await routerAdministrationClient.createQueue(queueRequest.id!, queueRequest);
+
+  const classificationPolicyRequest: ClassificationPolicy = {
+    id: "classification-policy-123",
+    name: "test-policy",
+    fallbackQueueId: "queue-123",
+    queueSelectors: [
+      {
+        kind: "conditional",
+        labelSelectors: [
+          {
+            key: "foo",
+            labelOperator: "equal",
+            value: { "default": 10 }
+          }
+        ]
+      }
+    ],
+    prioritizationRule: {
+      kind: "static-rule",
+      value: { "default": 2 }
+    }
+  };
+
+  try {
+    const request = classificationPolicyRequest;
+
+    const result = await routerAdministrationClient.createClassificationPolicy(request.id!, request);
+
+    console.log("classification policy: " + result);
+  } catch (error) {
```

I don't think we should catch and swallow the error. I'd remove the try/catch and let it throw.

---

This comment applies to all samples